transports/msmq/routing-extensibility_content_msmqtransport_[1,).partial.md

```diff
@@ -0,0 +1,11 @@
+## Custom instance mapping
```

Comment:
@timbussmann what was the better title that you came up with when we discussed this?

Reply:
something like "instance mapping extensibility" maybe?